test/cql-pytest/test_tablets.py

```diff
@@ -121,3 +121,49 @@ def test_tablets_are_dropped_when_dropping_table(cql, skip_without_tablets):
             assert res.one().tablet_count > 0, f"table {keyspace_name}.{table_name}: zero tablets allocated"
         res = cql.execute(f"SELECT * FROM system.tablets WHERE keyspace_name='{keyspace_name}' AND table_name='{table_name}' ALLOW FILTERING")
         assert not res, f"table {keyspace_name}.{table_name} was found in system.tablets after the table was dropped: tablet_count={res.one().tablet_count}"
+
+
+def test_tablets_are_dropped_when_dropping_view(cql, skip_without_tablets):
+    ksdef = "WITH REPLICATION = {'class' : 'NetworkTopologyStrategy', 'replication_factor' : '1'} AND TABLETS = {'initial': 1}"
+    with new_test_keyspace(cql, ksdef) as keyspace:
+        keyspace_name = keyspace
+        table_name = ""
+        view_name = ""
```

Comment:
Again all these variable-settings/renaming/copying are unecessary ands just make the test harder to understand.

Reply:
fixed in next version